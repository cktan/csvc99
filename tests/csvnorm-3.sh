# Test Case : CSV with double quotes in fields
../csvnorm in/csvnorm-3.csv 
