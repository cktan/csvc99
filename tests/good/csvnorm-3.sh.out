"Jane ""the great"" Doe",Smith,25
